backend/internal/proxy-host.js

```diff
@@ -406,7 +405,6 @@ const internalProxyHost = {
 					.where('is_deleted', 0)
 					.groupBy('id')
 					.omit(['is_deleted'])
-					.allowEager('[owner,access_list,certificate]')
```

Comment:
Note to Self until I can get to a place where I can fix the code: Need to keep the 'owner' and 'certificate' lines here or add them to the query builder. Also, we will need to use `allowGraph` as `allowEager` is will reset the allowed list (and its deprecated).